lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -0,0 +1,45 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that before/after(:all) isn't being used
+      # as they are not rolled back.
+      # See https://relishapp.com/rspec/rspec-rails/docs/transactions
```

Comment:
We could link to this discussion: https://github.com/rspec/rspec-core/issues/573